partiql-conformance-test-generator/Cargo.toml

```diff
@@ -23,6 +23,8 @@ edition = "2021"
 
 [dependencies]
 walkdir = "2.3"
-ion-rs = "0.6.0"
-codegen = "0.1.3"
+ion-rs = "0.14.*"
+codegen = "0.2.*"
 Inflector = "0.11.4"
+miette = "5.*"
+thiserror = "1.*"
```

Comment:
What are these two crates used for? I don't believe `thiserror` was used in this PR. I saw `miette` was used in the `Result` output types for some functions along with the `IntoDiagnostic` trait.

Reply:
thiserror is currently unused
miette is imported for ease of error reporting: https://docs.rs/miette/latest/miette/

the cli uses these currently